src/testing.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @overview Provides instances of shescape for testing purposes.
+ * @license MPL-2.0
+ */
+
+import { isStringable, toArrayIfNecessary } from "./reflection.js";
+
+/**
+ * A simple replacement for shescape that has the same input-output profile as
+ * the real shescape implementation.
```

Comment:
Make explicit that this is meant for testing to avoid misuse:

```suggestion
 * A test stub of shescape that has the same input-output profile as the real
 * shescape implementation.
```